daemon/cmd/identity.go

```diff
@@ -92,3 +95,22 @@ func (c cachingIdentityAllocator) AllocateCIDRsForIPs(ips []net.IP, newlyAllocat
 func (c cachingIdentityAllocator) ReleaseCIDRIdentitiesByID(ctx context.Context, identities []identity.NumericIdentity) {
 	c.ipcache.ReleaseCIDRIdentitiesByID(ctx, identities)
 }
+
+func (c *cachingIdentityAllocator) ReleaseCIDRs(ctx context.Context, prefixes []netip.Prefix) {
+	for _, prefix := range prefixes {
+		lbls := cidr.GetCIDRLabels(prefix)
+		id := c.LookupIdentity(ctx, lbls)
+		if id == nil {
+			log.WithFields(logrus.Fields{
+				logfields.CIDR: prefix.String(),
+			}).Warning("Restored Identity for CIDR cannot be found")
```

Comment:
Should these be either errors or info? Warning is unspecific, and it seems kinda bad if this happens?

Reply:
I think you're right. A while back Louis and I tried to collect some feedback on thresholds for using different log levels, and [these guidelines](https://docs.google.com/document/d/142qQ81YQbYysJ_GPQJtVRRnZ-zFCKH_2NsQ-FEnlK7Y/edit#bookmark=id.bsw8c4e8mrss) are where we got to. At some point we should formalize these, but the implications here could be that traffic to certain IPs is inhibited. I think that this could be an error with a prompt for the user to report the bug.

Reply:
I'll wrap this in a sync.Once too, there's no need to spam the logs for every IP.